Resources/Prototypes/_LostParadise/Entities/Objects/Specific/Medical/hypospray.yml

```
@@ -23,3 +23,81 @@
+    state: borghypo_s
+  - type: Item
+    sprite: Objects/Specific/Medical/hypospray.rsi
+  - type: SolutionContainerManager
+    solutions:
+      hypospray:
+        maxVol: 50
+  - type: SolutionRegeneration
+    solution: hypospray
+    generated:
+      reagents:
+      - ReagentId: Bicaridine
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)